terraform/stacks/cloudfront/staging.tf

```diff
@@ -0,0 +1,6 @@
+module "staging_cloudfront" {
```

Comment:
Do we need a new `terraform/stacks/cloudfront`? We already have `terraform/stacks/external` which [manages deploying resources to our commercial accounts](https://github.com/cloud-gov/cg-provision/blob/main/README.md?plain=1#L38)

So we could just add this module to `terraform/stacks/external/stack.tf` and it will get auto-deployed to create dev, stage, and prod resources in our commercial accounts from pipeline jobs like [this one](https://github.com/cloud-gov/cg-provision/blob/main/ci/pipeline.yml#L77)

Reply:
I'm strongly in favor of this idea, but I didn't address it here because of the cross-partition state dynamic we talked about a few days ago. I'd like to merge the external + dns + cloudfront stacks all together, but I'm hesitant to make a change to the external stack's state. It currently only talks to Commercial, I think — no reading GovCloud state.

I can look into it if you feel strongly though! One concern I have by having them separate is that Terraform won't be able to build its dependency graph properly and there will be ordering issues when something changes.